src/Kunstmaan/MediaBundle/AdminList/MediaAdminListConfigurator.php

```diff
@@ -113,12 +120,24 @@ public function canDelete($item)
     /**
      * @return int
      */
-    public function getLimit()
+    public function getLimit() : int
     {
-        return 250;
+        return $this->limit;
     }
 
     /**
+     * @param int $limit
+     * @return MediaAdminListConfigurator
+     */
+    private function setLimit(int $limit): MediaAdminListConfigurator
```

Comment:
Same here

Reply:
This is branching from 5.0, which is targeting PHP 7. I asked @Numkil before I committed.

Reply:
Ok, I understand but in composer we have version PHP >=5.6.0 so this case create BC break. In milestones on version 6.0 (#1445) we start using PHP 7.1 version skipped PHP 7.0 :)

Reply:
ok i'll remove the return types then :-)

Reply:
Thank You, nice work! :)